Modules/_posixsubprocess.c

```diff
@@ -594,6 +594,11 @@ child_exec(char *const exec_array[],
         POSIX_CALL(setsid());
 #endif
 
+#ifdef HAVE_SETPGID
+    if (pgid_to_set >= 0)
+        POSIX_CALL(setpgid(0, pgid_to_set));
```

Comment:
Please add braces (PEP 7).

Reply:
In this case I'm being consistent with all of the surrounding code.